src/app/components/Headline/index.test.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import Headline from './index';
 import { textBlock } from '../../models/blocks';
-import {shouldMatchSnapshot} from '../../helpers/tests/testHelpers';
+import {shouldMatchSnapshot, isNull} from '../../helpers/tests/testHelpers';
```

Comment:
Surprised linting isn't complaining about this. A trivial change: add spacing to this destructure, e.g. `{ shouldMatchSnapshot, isNull }`